.travis/install-gambit.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+if [ ! -f $HOME/gambit/bin/gsi ]; then
+  wget http://www.iro.umontreal.ca/~gambit/download/gambit/v4.8/source/gambit-v4_8_9-devel.tgz
+  tar xzvf gambit-v4_8_9-devel.tgz
+  pushd gambit-v4_8_9-devel
+  ./configure --prefix=$HOME/gambit
+  make
+  make install
+  make bootstrap
```

Comment:
do we need this after make install? seems redundant.

Reply:
I don't know, I can try without it